corehq/apps/cloudcare/static/cloudcare/js/form_entry/utils.js

```diff
@@ -89,7 +89,11 @@ hqDefine("cloudcare/js/form_entry/utils", function () {
         // Must add the form-control class to the input created by mapbox in order to edit.
         var inputEl = $('input.mapboxgl-ctrl-geocoder--input');
         inputEl.addClass('form-control');
-        inputEl.on('keydown', _.debounce(self._inputOnKeyDown, 200));
+        inputEl.on('keydown', _.debounce((e) => {
+            if (e.keyCode === 38 || e.keyCode === 40) {
```

Comment:
nit: good to have a comment about what these keys are so that one doesn't need to find for it if needed

Reply:
and does this remove the call to `_inputOnKeyDown`?

Reply:
[b830161](https://github.com/dimagi/commcare-hq/pull/32637/commits/b830161eca8dccbc632dba52804ed47c44f56a4f) using event.key with key names which are more descriptive than keyCodes
[80876ac](https://github.com/dimagi/commcare-hq/pull/32637/commits/80876ac322cfcd94150b5c259326186256b34c86) fixed the inputOnKeyDown call